lua/tests/infoview_lsp_spec.lua

```diff
@@ -0,0 +1,62 @@
+local infoview = require('lean.infoview')
+local helpers = require('tests.helpers')
+
+local function get_info_lines(infoview_info)
+  return table.concat(vim.api.nvim_buf_get_lines(infoview_info.bufnr, 0, -1, true), "\n")
```

Comment:
This seems useful enough to even be in the real code rather than just a test helper (as like `infoview.get_text` or something?)